src/main/java/racingcar/model/WinnerCars.java

```diff
@@ -0,0 +1,34 @@
+package racingcar.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class WinnerCars {
+    //<필드>
+    private List<Car> winningCars;
+
+    //<생성자>
+    public List<Car> getWinningCars() {
+        return winningCars;
+    }
+
+    //<메소드>
+    //우승차 리스트 만들기
+    public WinnerCars(List<Car> carList) {
+        //max값 찾기
+        int max = 0;
+        for (Car car : carList) {
+            if (car.getPosition() >= max) {
+                max = car.getPosition();
+            }
+        }
```

Comment:
```suggestion
        int max = carList.stream()
                .mapToInt(Car::getPosition)
                .max()
                .orElse(0);
```

P2)
이 부분은 for문 대신 stream을 사용해서 좀 더 가독성 있게 바꿀 수 있어요.
아래 코드도 바꿔볼까요??
힌트는 `filter`를 사용하면 됩니다

Reply:
수정했습니다!
stream은 처음 들어보는 개념이였는 데 데이터 속에서 필요한 값을 가공하여 제공한다는 점이 매우 흥미로웠습니다.
다음 미션 때부터 적극적으로 활용해보도록 하겠습니다!